taui/src/components/top-bar.js

```diff
@@ -71,14 +75,35 @@ export default class TopBar extends PureComponent<Props, State> {
     )
   }
 
+  loadingListingsButton (props) {
+    const { message } = props
+    return (
+      <div style={{ display: 'inline-block', position: 'relative' }}>
```

Comment:
Haven't tried it locally but I'm pretty sure you can drop the wrapping `div` and just add `position: relative` to `&__button-loading` in `_top-bar.scss`